### PR TITLE
Create 2023-08-10-warehouse-connectors.mdx

### PR DESCRIPTION
**Pull request by tiffany-mixpanel**
Adding page for warehouse connectors GA

### CODE REVIEW

File: pages/changelogs/2023-08-10-warehouse-connectors.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </iframe>
+</div>
+
+Today, all customers have access to Warehouse Connectors, which allows you to natively import data from Snowflake or BigQuery into Mixpanel. With this feature, you can set up recurring syncs from your data warehouse and ensure that Mixpanel is always in sync with your trusted data.
+
+This native connection not only simplifies event implementation since you no longer need engineering resources to manually track events, but it also makes it easier to analyze all types of company data in Mixpanel and gain a holistic view of your customers.
+
+For example, you can bring into Mixpanel:
+
+- **AppDB tables**: Signups, Reports Created, Dashboards Created
```
Comment (ranic) [inline pages/changelogs/2023-08-10-warehouse-connectors.mdx]: "AppDB" -> Application Database

File: pages/changelogs/2023-08-10-warehouse-connectors.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- **Salesforce data**: Opportunity Created, Account Created, Account Upsell
+- **Support data**: Ticket Created, Ticket Resolved
+- **Billing/Consumption data:** Ad Spend, Units Consumed, etc.
```
Comment (ranic) [inline pages/changelogs/2023-08-10-warehouse-connectors.mdx]: I'd pull "Ad Spend" into its own category. It's very different than billing and is a big use case on its own.

File: pages/changelogs/2023-08-10-warehouse-connectors.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        webkitallowfullscreen mozallowfullscreen allowfullscreen
+        style={{position: 'absolute', 'top': 0, 'left': 0, 'width': '100%', 'height': '100%'}}>
+    </iframe>
+</div>
+
+Today, all customers have access to Warehouse Connectors, which allows you to natively import data from Snowflake or BigQuery into Mixpanel. With this feature, you can set up recurring syncs from your data warehouse and ensure that Mixpanel is always in sync with your trusted data.
+
+This native connection not only simplifies event implementation since you no longer need engineering resources to manually track events, but it also makes it easier to analyze all types of company data in Mixpanel and gain a holistic view of your customers.
+
+For example, you can bring into Mixpanel:
```
Comment (ranic) [inline pages/changelogs/2023-08-10-warehouse-connectors.mdx:22]: Sentence structure is a bit off

File: pages/changelogs/2023-08-10-warehouse-connectors.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- **AppDB tables**: Signups, Reports Created, Dashboards Created
+- **Salesforce data**: Opportunity Created, Account Created, Account Upsell
+- **Support data**: Ticket Created, Ticket Resolved
+- **Billing/Consumption data:** Ad Spend, Units Consumed, etc.
+- **Clickstream data from Snowplow:** Views, clicks, etc
+
+And answer questions that are:
```
Comment (ranic) [inline pages/changelogs/2023-08-10-warehouse-connectors.mdx]: "This lets you answer questions like"

File: pages/changelogs/2023-08-10-warehouse-connectors.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- **Revenue-related:** What is % of sales opportunities do we win? What is the average deal size in North America?
```
Comment (ranic) [inline pages/changelogs/2023-08-10-warehouse-connectors.mdx]: I'd maybe remove the categories here and just list the questions

**Review by ranic — APPROVED** [review]
LGTM with a few suggested edits

**Review by tiffany-mixpanel — COMMENTED** [review]
made the edits

**Pull request merged**